Patch by schwardt@univention.de (Thu Apr 4 16:54:54 2013 +0200):

The python bindings may not call the destructor directly because memory is 
freed before all QT threads have stopped. This caused a SIGSEGV (Bug #27534).

Python now transfers the responsibility to QT for shutting down the
ItalcVncConnection object and freeing its memory. To initiate a shutdown
from pythons side, the function stop() has to be called which ends the
internal thread main loop run(). If run() finishes, it advises QT to clean 
the object up later on (via deleteLater()).


--- italc-2.0.0.ORIG/lib/src/ItalcVncConnection.cpp
+++ italc-2.0.0/lib/src/ItalcVncConnection.cpp
@@ -523,6 +522,7 @@
 	{
 		doConnection();
 	}
+	deleteLater();
 }
 
 
